client/main.lua

```diff
@@ -69,6 +71,10 @@ BodyParts = {
     ['RFOOT'] = { label = Lang:t('body.right_foot'), causeLimp = true, isDamaged = false, severity = 0 },
 }
 
+RegisterNetEvent('QBCore:Player:SetPlayerData', function(data)
+    PlayerData = data
```

Comment:
Add `if GetInvokingResource() then return end` above this to prevent a client triggering the event

Reply:
Done. Can you explain the logic? I'm guessing the server isn't considered an invoking resource? for this native?

Reply:
Should all net event handlers not have this check then?

Reply:
GetInvokingResource() only returns a value if the event is triggered from the same side, since QBCore:Player:SetPlayerData is triggered from the server, we do not want any clients to trigger it